librocksdb-sys/build.rs

```diff
@@ -140,6 +140,12 @@ fn try_to_find_lib(library: &str) -> bool {
         _ => "UNKNOWN"
     };
 
+    match env::var(format!("{}_BUILD", lib_name)).ok() {
```

Comment:
Should we use a constant here (and probably elsewhere) for easier discoverability (and mention this option in the docs)?